File: android/src/main/kotlin/com/imf/blade/BladeDelegate.kt
package com.imf.blade

import com.imf.blade.container.nativeEvents.PushFlutterPageEvent
import com.imf.blade.container.nativeEvents.PushNativePageEvent

interface BladeDelegate {
    fun pushNativePage(event: PushNativePageEvent)
    fun pushFlutterPage(event: PushFlutterPageEvent)
}

File: ios/Classes/protocol/BladeDelegate.swift
//
//  BladeDelegate.swift
//  blade
//
//  Created by sangya on 2021/8/31.
//

import Foundation
public protocol BladeDelegate {
    var navigationController:UINavigationController? { get set }
    func pushNativePage(event: FlutterBaseEvent)
    func pushFlutterPage(event: FlutterBaseEvent)
}
